service/src/utils/validateEnv.ts

```diff
@@ -34,7 +34,6 @@ function validateLibp2pEnvVars() {
     _tryReadEnv("TCP_PORT", process.env.TCP_PORT);
     _tryReadEnv("WS_PORT", process.env.WS_PORT);
     _tryReadEnv("SIGNAL_SERVER_LIST", process.env.SIGNAL_SERVER_LIST);
-    _tryReadEnv("DHT_PROTOCOL_PREFIX", process.env.DHT_PROTOCOL_PREFIX);
```

Comment:
I suppose we don't need this anymore? In which case we can go ahead and remove references to `DHT_PROTOCOL_PREFIX` everywhere else.

There _is_ a hardcoded `const PROTOCOL_PREFIX = "/archaeologist-service"` in `models/node-config`, and I doubt there's any reason to allow archs to optionally customise this